src/agent/agent.js

```diff
@@ -62,7 +62,7 @@ export class Agent {
             };
 
             if (init_message) {
-                this.handleMessage('system', init_message);
+                this.handleMessage('MINECRAFT', init_message);
```

Comment:
a little goofy, just makes strict formatting easier when the first and second messages aren't from system

Reply:
What happens with this when not using strict formatting? Is it sent as a system message or user? It seems like this would break.

Reply:
Strict formatting would've combined the system message with this init message. I am actually not sure if that's really a problem. I should probably just change this back, but I'm now away from my computer :(
Feel free to change back and test it out

Reply:
I meant what happens _without_ strict formatting. I'll take a look at it in a bit.

Reply:
Oh I think it's sent as a separate system message when not strictly formatted. I think part of the reason it responds so weirdly is because the messages are out of order and there are system roles mid chat, which strict formatting fixes. I saw a pretty dramatic improvement when I used strict formatting

Reply:
But if you're setting the role to `MINECRAFT` does that not mess things up? Unless somewhere else in the code it cahnges to `system` before sending. 

And ya. I'm honestly surprised that the openai models handle system messages mid-chat it as well as they do. We should compare "strict" vs. "unstrict" for openai models as well.

Reply:
I don't think so, it's just treated as a user named MINECRAFT, which I don't think would cause problems. 

Either way, I've removed it, not necessary for this pr.